adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAM2Vcf.scala

```diff
@@ -56,6 +56,10 @@ class ADAM2VcfArgs extends Args4jBase with ParquetArgs {
   @Args4jOption(required = false, name = "-single", usage = "Save as a single VCF file.")
   var single: Boolean = false
 
+  @Args4jOption(required = false, name = "-disable_fast_concat",
+    usage = "Disables the parallel file concatenation engine.")
+  var disableFastConcat: Boolean = false
```

Comment:
Not a fan of double-negatives; is there a better way to do args4j flags that default to true?